packages/runner-sdk/lib/action.ts

```diff
@@ -75,6 +76,7 @@ export abstract class NangoActionBase<
         this.providerConfigKey = config.providerConfigKey;
         this.runnerFlags = config.runnerFlags;
         this.activityLogId = config.activityLogId;
+        this.props = config;
```

Comment:
not a huge fan to store the entire props. We are gonna have logic that is going to use the class attributes and other that are going to use the props. We are also storing info that is not relevant to the class. 
Can we have a `this.shouldSampleLogs = props.scriptType === 'sync'` attribute instead?